release-centos7-llvm/gen_coverage.sh

```diff
@@ -54,7 +54,7 @@ cd -
 ## TODO: Add more testing binaries
 
 # Collect the prof raw data and generate cov report
-llvm-profdata merge -sparse "${COVERAGE_DIR}/*.profraw" -o "${COVERAGE_DIR}/merged.profdata"
+llvm-profdata merge -sparse ${COVERAGE_DIR}/*.profraw -o "${COVERAGE_DIR}/merged.profdata"
```

Comment:
why the `""` is removed?

Reply:
add `""` will not be viewed as regrex, cause "{COVERAGE_DIR}/*.profraw no such file or directory"